ui/app/adapters/auth-method.js

```diff
@@ -61,4 +61,8 @@ export default ApplicationAdapter.extend({
   exchangeOIDC(path, state, code) {
     return this.ajax(`/v1/auth/${encodePath(path)}/oidc/callback`, 'GET', { data: { state, code } });
   },
+
+  tune(path, data) {
+    return this.ajax(`${this.url(path)}tune`, 'POST', { data });
+  },
```

Comment:
This was added to replace the functionality from [ember-api-actions](https://github.com/mike-north/ember-api-actions) which was removed.